R/dygraph.R

```diff
@@ -3,10 +3,11 @@
 #' R interface to interactive time series plotting using the 
 #' \href{http://dygraphs.com}{dygraphs} JavaScript library.
 #' 
-#' @param data Time series data (must be an \link[xts]{xts} object or an object 
-#'   which is convertible to \code{xts}).
-#' @param periodicity Periodicity of data (automatically detected via
-#'   \link[xts:periodicity]{xts::periodicity} if not specified).
+#' @param data Either time series data or numeric data. (For time series, this
+#'   must be an \link[xts]{xts} object or an object which is convertible to
+#'   \code{xts}. For numeric data, this must be a named list or data frame)
```

Comment:
Maybe clarify the expected format and behavior for lists and data frame inputs (e.g. the first column provides the x values and subsequent columns provide 1 or more series of y values)